features/ubuntu_pro_fips.feature

```diff
@@ -105,7 +106,110 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO fips image
            | xenial  | libkrad0  | jq       | https://esm.ubuntu.com/fips/ubuntu xenial/main | fips                |
            | bionic  | libkrad0  | bundler  | https://esm.ubuntu.com/fips/ubuntu bionic/main | azure-fips          |
 
-    @series.lts
+    @series.focal
```

Comment:
You are splitting the tests because of the `openssh hmac` package checks right ?

If that is the case, I think we should follow the split performed on `enable_fips_cloud.feature` where we have test scenarios for the package checks and one that checks the fips kernel for each cloud

Reply:
Yes that is the reason for the splitting. Oh, let me go through the `enable_fips_cloud.feature` file.